lua/framework/physics/lua_material.cc

```diff
@@ -294,7 +293,7 @@ MatSetProperty(lua_State* L)
         auto G = LuaArg<int>(L, 4);
         auto sigma_t = LuaArg<double>(L, 5);
 
-        prop->MakeSimple0(G, sigma_t);
+        prop->Initialize(G, sigma_t);
       }
       else if (operation_index == static_cast<int>(OpType::SIMPLEXS1))
```

Comment:
the name SIMPLEXS1 is cryptic. Better name ?

Reply:
From my review:

> The name here is a bit tricky because the XS are not in a standard form that has a common name. It may be good to add a DOWN_SCATTERER option and call this FULL_SCATTERER, UPDOWN_SCATTERER, or MIXED_SCATTERER or something like that.

Reply:
Resolved with MODEL1G.